R/fstadv.R

```diff
@@ -237,17 +237,17 @@ fstadv_forecasts <- function(content, key, adv, adv_date) {
   # 96 and 120 hours). Some text products may have no forecasts at all (if the
   # storm is expected to degenerate or already has).
   forecasts <-
-    content %>%
-    stringr::str_match_all(pattern = ptn) %>%
+    content |>
+    stringr::str_match_all(pattern = ptn) |>
     # # Get only the columns needed excluding the matched string
-    # purrr::map(`[`, , 2:22) %>%
+    # purrr::map(`[`, , 2:22) |>
     # If any storm has 0 forecasts (i.e., the list element is empty), populate
```

Comment:
I think you nee to stop the pipe chain at 241, then do the 243 stuff, then restart at 243.

Reply:
It looks like line 243 is commented out, so it won't be compiled.

Should 241 pipe in to 246?

Reply:
I went through the code and found a number of places that were subsetting using `[]` so in the end I don't think it is just here ... but then again whatever you did seemed to work.  Take a look at my other branch called pipe2 #173